db/migrate/20230119181721_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
+
+    add_reference :comments, :user, foreign_key: true
```

Comment:
- [ ] You can add a reference for the `author` here because the comments belong to the author
e.g
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```